std/stdio.d

```diff
@@ -930,7 +930,7 @@ $(D rawRead) always reads in binary mode on Windows.
     {
         static import std.file;
 
-        auto testFile = testFilename();
+        auto testFile = std.file.deleteme();
```

Comment:
Not a huge fan of this, but `deleteme` is the "defacto" standard within Phobos and used in many other public examples.